### DISCUSSION TITLE
ico decoder

### DISCUSSION THREAD

**Discussion by KANekT** [SixLabors/ImageSharp#2782] [Ideas] [answered]
![image](https://github.com/user-attachments/assets/f36c54a5-2163-43a0-9cd0-3ad46f357d82)

get info for ico - height / width

using var image = Image.Load(bytes);

has error : "Image cannot be loaded. Available decoders:\n - PNG : PngDecoder\n - TGA : TgaDecoder\n - BMP : BmpDecoder\n - QOI : QoiDecoder\n - JPEG : JpegDecoder\n - Webp : WebpDecoder\n - PBM : PbmDecoder\n - GIF : GifDecoder\n - TIFF : TiffDecoder\n"

**Comment by JimBobSquarePants** [comment, marked as answer]
v4 will have ICO and CUR decoders and encoders. https://github.com/SixLabors/ImageSharp/pull/2579
